packages/astro/src/content/vite-plugin-content-virtual-mod.ts

```diff
@@ -360,7 +372,7 @@ export async function generateLookupMap({
 					const contentEntryType = contentEntryConfigByExt.get(extname(filePath));
 					if (!contentEntryType) throw UnexpectedLookupMapError;
 
-					const { id, slug: generatedSlug } = await getContentEntryIdAndSlug({
+					const { id, slug: generatedSlug } = getContentEntryIdAndSlug({
```

Comment:
this is an unrelated change, but this `await` is on a sync function, so I removed.